docs/UserGuide.md

```diff
@@ -40,14 +48,16 @@ clean Graphical User Interface (GUI) for easy comprehension of expenditure and s
    open the help window.<br>
    Some example commands you can try:
 
```

Comment:
Should we add a info block here to let users know of the notations/syntax we use? Such as code blocks etc.

Reply:
i think these were mentioned earlier command format portion, but we can add another code block for each sample commands to make it clearer!